### PR TITLE
Add release docs for v0.6.0

### PR DESCRIPTION
**Pull request by benjaminhuo**
Signed-off-by: Benjamin Huo <benjamin@kubesphere.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,40 +17,50 @@
...
+- Add buildah, kaniko, ko support, user can build apps with Dockerfile using buildah or kaniko, and build go application with ko now [#170](https://github.com/OpenFunction/OpenFunction/pull/170) [#171](https://github.com/OpenFunction/OpenFunction/pull/171)
+- Add OpenFunction ClientSet [#176](https://github.com/OpenFunction/OpenFunction/pull/176)
+- Support to keep build history [#179](https://github.com/OpenFunction/OpenFunction/pull/179)
+- Support to add labels and annotations to function workloads [#181](https://github.com/OpenFunction/OpenFunction/pull/181)
 
 ### Enhancement
 
-- Delete old serving after new serving running. #107
-
+- Depreciate the old install/uninstall scripts in favor of the cli tool [ofn](https://github.com/OpenFunction/cli/releases), now users can use [ofn](https://github.com/OpenFunction/cli/releases) to install/uninstall/demo OpenFunction
```
Comment (FeynmanZhou) [inline CHANGELOG.md:60]: Depreciate => Deprecated

**Review by tpiperatgod — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (benjaminhuo) [reply CHANGELOG.md:60]: good cache

Comment (Felixnoo) [reply CHANGELOG.md]: I see most verbs using the simple present tense. Suggest using the same tense here, namely `Deprecate`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,40 +17,50 @@
...
+- Now OpenFunction is compatible with K8s 1.17 ~ 1.20+, thanks to [ofn](https://github.com/OpenFunction/cli/releases)
+- Optimized Function status to reflect serving workload status more accurately [151](https://github.com/OpenFunction/OpenFunction/pull/151)
```
Comment (Felixnoo) [inline CHANGELOG.md:62]: `Optimized` -> `Optimize`
Suggest using the simple present tense.

File: CHANGELOG.md (hunk 1 above)
Comment (benjaminhuo) [reply CHANGELOG.md]: already fixed

File: CHANGELOG.md (hunk 2 above)
Comment (benjaminhuo) [reply CHANGELOG.md]: fixed

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**